dotcom-rendering/src/web/components/MostViewedFooterData.importable.tsx

```diff
@@ -15,13 +15,22 @@ interface Props {
 	sectionName?: string;
 	format: ArticleFormat;
 	ajaxUrl: string;
+	edition: string;
 }
 
-function buildSectionUrl(ajaxUrl: string, sectionName?: string) {
+function buildSectionUrl(
+	ajaxUrl: string,
+	edition?: string,
```

Comment:
There's an [editionId type](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/lib/edition.ts), which I think is what gets used by e.g. the layout components - would it make sense to use that here?